imports/api/contracts/contracts.js

```diff
@@ -94,6 +94,16 @@ Contracts.helpers({
     if (this.delegateId) return Partners.findOne(this.delegateId);
     return undefined;
   },
+  entitledToView() {
+    if (!this.partnerId) return undefined;
+    let partnerIds = [this.partnerId];
+    if (this.relation === 'member' && this.parcelId) {
+      const Memberships = Mongo.Collection.get('memberships');
+      const owners = Memberships.findActive({ role: 'owner', parcelId: this.parcelId }).map(m => m.partnerId);
```

Comment:
Itt az aktiv tulajokat gyujtjuk ossze. De nem biztos hogy a contract egy aktiv contract. Ha ez egy regi contract, akkor az akkori tulajdonosokat kéne összeszedni.
Mivel ez egy borzalmasan bonyolult dolog lenne. Mi  lenne, ha csak akkor vegeznenk el ezt a tulajdonso összeszedést, ha a contract aktív. Ha nem akkor meg nem foglalkoznánk vele.

Reply:
Igen, ezt nem defináltuk végül, hogy ki láthatja az aktív meg a korábbi parcel contractokat ill. az ahhoz tartozó tx-eket. Van a partner-history-ban olyan funkció, hogy a jogelőd kontrakt tx-eit is meg lehet tekinteni. Azt akkor hivatalosan csak az láthatja, aki a kontrakt partner volt?

Reply:
Na az egy masik erdekes kerdes. Nem tudom azt eddig hogyan tudta latni.